demos/speech_recognition_offline_demo/python/speech_recognition_offline_demo.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import librosa
+import numpy as np
+import scipy
+import wave
+
+from argparse import ArgumentParser, SUPPRESS
+from openvino.inference_engine import IECore
+
+
+class QuartzNet:
+    pad_to = 16
+    alphabet = " abcdefghijklmnopqrstuvwxyz'"
+
+    def __init__(self, ie, xml_path, input_shape, device):
+        assert not input_shape[2] % self.pad_to, f"{self.pad_to} must be a divisor of input_shape's third dimension"
+        self.ie = ie
+        net = self.ie.read_network(xml_path)
+        if len(net.input_info) != 1:
+            raise RuntimeError('QuartzNet must have one input')
+        encoder_input_shape = next(iter(net.input_info.values())).input_data.shape
+        if len(encoder_input_shape) != 3:
+            raise RuntimeError('QuartzNet input must be 3-dimensional')
+        if encoder_input_shape[1] != input_shape[1]:
+            raise RuntimeError("QuartzNet input second dimension can't be reshaped")
+        if encoder_input_shape[2] % self.pad_to:
+            raise RuntimeError(f'{self.pad_to} must be a divisor of QuartzNet input third dimension')
+        if len(net.outputs) != 1:
+            raise RuntimeError('QuartzNet must have one output')
+        decoder_output_shape = next(iter(net.outputs.values())).shape
+        if len(decoder_output_shape) != 3:
+            raise RuntimeError('QuartzNet decoder output must be 3-dimensional')
+        if decoder_output_shape[2] != len(self.alphabet) + 1:  # +1 for blank char
+            raise RuntimeError('QuartzNet decoder second dimension must be similar to a length of an alphabet assumed')
```

Comment:
This doesn't mean anything to the user. What is "an alphabet assumed" and what does "similar" mean?

Reply:
The new version is better, but the user still wouldn't know what the expected value is. Why not just use the number?

Reply:
Updated